guide/daily-spending-wallet/requesting/receiving.md

```diff
@@ -0,0 +1,199 @@
+
+{% include tip/open.html color="blue" icon="info" label="Experimental options" %}
+
+[Peerswap](https://www.peerswap.dev/) or [Collaborative funding](https://bitcoinops.org/en/topics/dual-funding/) may be more trustless options that do not require a third-party LWS to receive and send on-chain payments. Though these are not currently widely adopted.
+
+{% include tip/close.html %}
+
+### Refunds
+
+Refunds are required in some scenarios when a submarine swap can not be completed successfully. A refund is when a swap fails and the incoming on-chain bitcoin is now in control by the LWS. Users will need to claim these on-chain funds as a refund.
+
+Reasons for refunds will vary based on how the LWS swap service is implemented but below are some common scenarios:
+
+- LWS swap service is offline
+- Receiving more than the users inbound capacity 
+- Re-using a swap-address to receive an on-chain payment
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain@2x.png"
+   layout = "shadow"
+   caption = "If a submarine swap fails, notify the user that they need to claim the refunded payment."
+   alt-text = "Screen showing a warning error notification prompting the user to claim a refund of their failed submarine swap."
+   width = 250
+   height = 542
+   modalWidth = 250
+   modalHeight = 542
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity@2x.png"
+   layout = "shadow"
+   caption = "The activity page is another place where you can indicate to a user that they need to claim their refund."
+   alt-text = "Screen showing the activity menu with an entry for the user to claim their refunded payment."
+   width = 250
+   height = 542
+   modalWidth = 250
+   modalHeight = 542
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund@2x.png"
+   layout = "shadow"
+   caption = "Make it simple for a user to claim their refund."
+   alt-text = "Screen showing a form to claim the refund to an on-chain address."
+   width = 250
+   height = 542
+   modalWidth = 250
+   modalHeight = 542
+%}
+
+</div>
+
+## Received payment
+
+Received payments to this wallet are clearly indicated to the user. Fees paid, if any, are clear to the user when viewing the payments details. Users can save and share a receipt.
```

Comment:
```suggestion
When this wallet receives a payment, it notifies the user. If any fees are paid, they are clear to the user when viewing the payment details. Users can save and share a receipt.
```
I find the wording to be a little redundant. "Clear to the user" is used twice in a row.